api/VERSION

```diff
@@ -1 +1 @@
-v5
+v6
```

Comment:
Add a Makefile target to regex replace to replace? I see several patterns which we can do.

Reply:
Good idea.  we can start with a script to do the replacement when changing version.  I will do that with a separate PR.